tests/gtests/test_SW_Flow_Lib.cc

```diff
@@ -165,17 +165,19 @@ TEST(SWFlowTest, SWFlowSaturatedPercolation) {
 
     // declare inputs
     double pptleft = 5.0;
-    double standingWater;
+    double standingWater = 0.;
     double drainout;
+    const double swc0init = 0.8;
 
     // ***** Tests when nlyrs = 1 ***** //
     ///  provide inputs
     int nlyrs = 1;
-    double swc[1] = {0.8};
+    double swc[1];
```

Comment:
On my end, the command `CXX=g++ make clean test_run` results in this test to fail. If I initialize the variable `swc` to `{0}`, the test passes. Is this reproducible?

Reply:
thanks! Commit 861950ef074f6254079a1b1f582376efb876326d aims to address this. This didn't show up for me with g++ v14.2.0

Reply:
That's interesting, I was using g++ v12.4.0, however, it is now gone!